website/static/js/pages/wiki-edit-page.js

```diff
@@ -167,3 +168,25 @@ $(document).ready(function () {
     // Tooltip
     $('[data-toggle="tooltip"]').tooltip();
 });
+
+var $comments = $('.comments');
+if ($comments.length && window.contextVars.wiki.wikiID !== 'None') {
+    var currentUser = {
```

Comment:
Do you need to reserialize `currentUser`? Is there a reason not to pass `window.contextVars.currentUser` to `Comment.init`?